apps/judicial-system/web/src/components/Table/DurationDate/DurationDate.tsx

```diff
@@ -0,0 +1,48 @@
+import { useIntl } from 'react-intl'
+import parseISO from 'date-fns/parseISO'
+import { tables } from '@island.is/judicial-system-web/messages'
+import { Text } from '@island.is/island-ui/core'
+import { Case, CaseState } from '@island.is/judicial-system/types'
+
+import { formatDate } from '@island.is/judicial-system/formatters'
+
+export function getDurationDate(
+  state: Case['state'],
```

Comment:
How come you decided to infer the types from Case here instead of explicitly defining them?